feat(docs): document FORCE_COLOR env var that allows disabling colored output (#1917)
hashicorp/terraform-cdk: website/docs/cdktf/create-and-deploy/environment-variables.mdx

| CDKTF_LOG_LEVEL                | Controls how much log information CDKTF prints to the console. You can set it to one of the following values: `all`, `debug`, `info`, `warn`, `error`, `fatal`, `off`.                                                        | `warn`                       |
| CHECKPOINT_DISABLE             | If set to any value, CDKTF will not send [telemetry](/cdktf/telemetry) data.                                                                                                                                                  |                              |
| DISABLE_VERSION_CHECK          | If set to `true` or `1`, CDKTF will not perform a version check. If it is not set, CDKTF verifies that your installed library version matches the CLI version. If they do not match, CDKTF displays a warning.                |                              |

> | FORCE_COLOR                    | If set to `0`, colored output will be disabled. See [`chalk` documentation](https://github.com/chalk/chalk#supportscolor) for other supported values.                                                                         |                              |

Reviewer: I'm assuming this is what we mean here? lmk if I'm wrong :) 

```suggestion
| FORCE_COLOR                    | If set to `0`, CDKTF disables colored CLI output. Refer to [`chalk` documentation](https://github.com/chalk/chalk#supportscolor) for other supported values.                                                                         |                              |
```
